### DISCUSSION TITLE
Important: How can I subscribe to every buy/sell txn of a coin?

### DISCUSSION THREAD

**Discussion by ajaythxkur** [aptos-labs/aptos-developer-discussions#505] [Questions] [answered]
I need to fetch every buy/sell txn of a coin, whenever someone swaps the coin. The thing is I don't know where to find the right websockets/APIs for this.

**Comment by 0x-j** [comment, marked as answer]
there's no websocket api now. You can build an indexer for that. We have a tutorial for that 

https://learn.aptoslabs.com/en/dapp-templates/custom-indexer-template
